docs/tutorials/index.rst

```diff
@@ -0,0 +1,49 @@
+GeoMesa |release| Tutorials
+===========================
+
```

Comment:
Carp.  I didn't English well.  

Do the tutorials require GeoMesa-ext and GeoMesa-tutorials to be public and all that coordinated?